vllm-tensorizer/Dockerfile

```diff
@@ -1,66 +1,57 @@
-ARG BASE_IMAGE="ghcr.io/coreweave/ml-containers/torch-extras:es-22.04-58a49a2-base-cuda12.1.1-torch2.1.2-vision0.16.2-audio2.1.2-flash_attn2.4.2"
-
-FROM scratch as freezer
+ARG BASE_IMAGE="ghcr.io/coreweave/ml-containers/torch-extras:es-compute-12.0-67208ca-nccl-cuda12.9.0-ubuntu22.04-nccl2.27.3-1-torch2.7.1-vision0.22.1-audio2.7.1-abi1"
```

Comment:
CI Should be passing base image as this will always be out of date.  This should move to something similar to what is done with the other torch images.  Also this looks like a non-main branch tag.  We need t o be building from main tags, not random branches.